ui/components/app/transaction-detail/index.scss

```
@@ -15,4 +15,55 @@
       text-transform: uppercase;
     }
   }
+
+  &-edit-V2 {
+    margin-bottom: 10px;
```

As a general rule all `padding` and `margin` values should follow an 8px grid. They should be divisible by `8px` with the exception being `4px`. So `4px`, `8px`, `16px`, `24px`, `32px`, `40px` etc. It's sometimes hard for the designers to get that spacing exact in their figma files but for us it's easy! If you come a cross a pixel value that is close. I would strongly suggest just rounding it up or down to the nearest number divisible by 8. This standardizes our spacing and will make components align better once we have full adoption.

[Reference in slack](https://consensys.slack.com/archives/G4V2HTG0Y/p1636139888050200?thread_ts=1636139616.050000&cid=G4V2HTG0Y)

```suggestion
    margin-bottom: 8px;
```